packages/react/src/components/scheduler-agenda/scheduler-agenda.tsx

```diff
@@ -0,0 +1,61 @@
+import './scheduler-agenda.scss';
+
+import React, { useCallback } from 'react';
+import { Duration } from 'luxon';
+import List from 'devextreme-react/list';
+import Button from 'devextreme-react/button';
+import { useScreenSize } from '../../utils/media-query';
+
+const getDurationString = (appointment) => {
+  const duration = Duration.fromMillis(appointment.endDate - appointment.startDate).rescale();
+  return duration.toFormat("h'h' m'm'");
+};
+
+const TimeContent = ({ appointment }) => {
+  const start = appointment.startDate?.toLocaleTimeString(undefined, {
+    hour: 'numeric',
+    minute: 'numeric',
+    hour12: false
+  });
+  return <div className='time'>
+    <div className='start'>{start}</div>
+    <div className='duration'>{getDurationString(appointment)}</div>
+  </div>;
+};
+
+export const SchedulerAgenda = ({ selectedAppointment = { startDate: new Date() }, toggleOpen, resources, items, showAppointmentPopup }) => {
+  const { isLarge } = useScreenSize();
+
+  const renderListItem = useCallback((item) => {
+    return <div className='list-item'>
+      <div className='time'>
```

Comment:
The div with the same className is rendered in the TimeContent component

Reply:
renamed list item in agenda to agenda-list-item